.github/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+#. AIRAC (2310) - Updated BEL VOR coordinates - thanks to @ChrisXPP (Christoph Reule)
+
 # Changes from release 2023/09a to 2023/10
 1. Bug - Corrected Scottish TMA Mentor (STC_M_CTR) callsign prefix for alternative ownership functionality - thanks to @rishab-alt
 2. Enhancement - Added ground network data for Bristol (EGGD) - thanks to @MikePike-collab (Mike Pike) & @luke11brown (Luke Brown)
```

Comment:
```suggestion
# Changes from release 2023/09a to 2023/10
1. Bug - Corrected Scottish TMA Mentor (STC_M_CTR) callsign prefix for alternative ownership functionality - thanks to @rishab-alt
2. Enhancement - Added ground network data for Bristol (EGGD) - thanks to @MikePike-collab (Mike Pike) & @luke11brown (Luke Brown)
3. AIRAC (2310) - Updated BEL VOR coordinates - thanks to @ChrisXPP (Christoph Reule)
```

As discussed on one of your previous PRs @ChrisXPP, this is the format that a changelog entry should take.